src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+    struct StrategyAndMultiplier {
+        IStrategy strategy;
+        uint96 multiplier;
+    }
+	struct RangePayment {
+        // Strategies & relative weights of shares in the strategies
+        StrategyAndMultiplier strategiesAndMultlipliers;
+        IERC20 token;
+        uint256 amount;
+        uint256 startTimestamp;
+        // duration in seconds
+        uint256 duration;
```

Comment:
Would it make sense to have this number be "weeks"? If it was, we wouldn't have to check the input and we could eliminate an error condition completely.

Reply:
I think it depends if we think the `calculationIntervalSeconds` is fixed (at one week) or will change.
using a duration in seconds + doing input sanitization would more fluidly support changing this interval; as long as the old new interval is equally divisible by the new interval, you get total backwards compatibility while enabling "new" (finer grain) control.
I'm OK switching this to weeks, just summarizing what I think is the best reasoning for not doing so.